ios-target/Networking/Services/TargetAPI.swift

```diff
@@ -40,4 +41,18 @@ class TargetAPI {
       failure(error)
     })
   }
+  
+  class func deleteTarget(_ targetId: Int, success: (() -> Void)? = nil, failure: ((_ error: Error) -> Void)? = nil) {
+    let url = targetsUrl + String(targetId)
+    APIClient.request(.delete, url: url, success: {_, _ in 
+      success?()
+    }, failure: { error in
+      // Dirty fix for backend error
+      if error is Alamofire.AFError {
+        success?()
```

Comment:
why call success here? maybe if the error is not from the backend, you can default to something generic like `Something went wrong`

Reply:
The problem is the backend should send a 204 which is a no content response but it sends a 200 so it detects an error ´Alamofire.AFError responseSerializationFailed´ 🤦🏽‍♀️. That's why it calls success when this is the error detected